include/aikido/util/detail/yaml_extension.hpp

```diff
@@ -135,13 +131,13 @@ struct convert<Eigen::
       throw YAML::RepresentationException(getMark(node), ss.str());
     }
 
-    if (node.Tag() == "!Vector")
+    if (node[0].Type() == YAML::NodeType::Scalar)
     {
       matrix.resize(rows, 1);
       for (Index i = 0; i < rows; ++i)
         matrix(i, 0) = node[i].template as<_Scalar>();
```

Comment:
:+1: I believe that case is covered by [this](https://github.com/personalrobotics/aikido/pull/200/files#diff-ee420700678d3f6c35f1b8428db1bd8eR163).

Another corner case would be when `node` doesn't have a child, which I'm not sure if we care about the case as well.